tests/metaworld/envs/mujoco/sawyer_xyz/test_hidden_goal_env.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from tests.helpers import step_env
+from metaworld.envs.mujoco.env_dict import ALL_V2_ENVIRONMENTS_GOAL_HIDDEN
+
+
+def test_hidden_goal_envs():
+    
+    for env_key, env_cls in ALL_V2_ENVIRONMENTS_GOAL_HIDDEN.items():       
+        
+        assert "goal-hidden" in env_key
+        assert "GoalHidden" in env_cls.__name__
```

Comment:
I believe there's a `string.startswith('...')` method if you want to make this test more expressive